node/http_test.ts

```diff
@@ -148,3 +148,150 @@ Deno.test("[node/http] request default protocol", async () => {
   });
   await promise;
 });
+
+Deno.test("[node/http] send request with non-chunked body", async () => {
+  const promise = deferred<void>();
+  let requestHeaders: Record<string, string> = {};
+  const requestBody: string[] = [];
+
+  const server = http.createServer((req, res) => {
+    req.on("data", (chunk) => {
+      requestBody.push(chunk.toString());
```

Comment:
Does it work to call `req.setEncoding('utf8')` and then concatenating the chunks in a string. Same comment for the other tests.

Reply:
Yes, it works fine in that case. Should I modify the tests?

Reply:
I think it's safer to do it that way (less assumptions about chunk sizes and more resilient to multibyte characters being split across chunks).

Reply:
I see. I have modified the unit tests. (7bd2bdf)